csharp/test/Ice/acm/TestI.cs

```diff
@@ -9,8 +9,7 @@ namespace ZeroC.Ice.Test.ACM
 {
     public class RemoteCommunicator : IRemoteCommunicator
     {
-        public IRemoteObjectAdapterPrx
-        createObjectAdapter(int timeout, int close, int heartbeat, Current current)
+        public IRemoteObjectAdapterPrx createObjectAdapter(int timeout, int? close, int? heartbeat, Current current)
```

Comment:
I think using `string?` for close and heartbeat would make more sense.

Reply:
fixed